### PR TITLE
Add Instant KO

### PR DESCRIPTION
**Pull request by wooferzfg**
Replacement for https://github.com/wooferzfg/wwrando/pull/5

### CODE REVIEW

**Comment by Crain-32** [comment]
@wooferzfg In classic Fashion, I'm back after two weeks with my testing.

Fails to randomize because `logic.py:54` doesn't include the Plando File as a constructor parameter. However it worked up to that point.

**Comment by wooferzfg** [comment]
@Crain-32 The issue should be fixed now

**Comment by Crain-32** [comment]
image
Got back around to this, just keeping you up to date. 

I haven't seen this error before, even with all the patches the Multiworld does. Going to look into it this weekend.
Build worked great though.

File: asm/patches/instant_ko.asm [hunk 1]
```diff
@@ -0,0 +1,85 @@
+.org @NextFreeSpace
```
Comment (Crain-32) [inline asm/patches/instant_ko.asm:1]: `.open "sys/main.dol"` Needs this at the top of the file, then rerun `assemble.py` in the ASM Folder.

Comment (Crain-32) [reply asm/patches/instant_ko.asm:1]: @wooferzfg I did the above in my local machine and it fixed the issue. I'll keep an eye out for the eye when it's built, but I've tested it with this fix locally, and all the instant kills worked.

The best was you can kill yourself with soup, and if you Save and Quit then you start the file with double damage and 3 hearts, I'm going to leave that in as a fun little bonus.

Comment (wooferzfg) [reply asm/patches/instant_ko.asm:1]: @Crain-32 Could you open a new PR based on my branch? That way you can make your own changes again

**Pull request closed without merging**